build.sbt

```diff
@@ -10,10 +9,20 @@ lazy val root = project.in(file("."))
     version := parsleyVersion,
     target in Compile in doc := baseDirectory.value / "docs",
 
-    libraryDependencies ++= Seq("org.scalactic" %% "scalactic" % "3.0.8" % "test",
-                                "org.scalatest" %% "scalatest" % "3.0.8" % "test"),
-    scalaVersion := scala2Version,
-    crossScalaVersions := List(scala2Version, "2.12.12"),
+    libraryDependencies ++= Seq(
+      "org.scalactic" %% "scalactic" % "3.2.2" % Test,
+      "org.scalatest" %% "scalatest" % "3.2.2" % Test
```

Comment:
Upgrade necessary because it uses macros, so we need a version published for Dotty.

Reply:
Yup, good by me!